examples/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.messagebird</groupId>
     <artifactId>examples</artifactId>
-    <version>3.0.1</version>
+    <version>3.0.2</version>
```

Comment:
no need to push pom.xml files I will update them when I will release the new version.